servers/ur/src/app.js

```diff
@@ -1,84 +1,43 @@
-import { Readable } from 'node:stream'
-import express from 'express'
-import WarpArBundles from 'warp-arbundles'
+import { join } from 'node:path'
 
-import { router } from './router.js'
+import cors from 'cors'
+import express from 'express'
+import heapdump from 'heapdump'
+import { pipe } from 'ramda'
 
-const { DataItem } = WarpArBundles
+import { config } from './config.js'
+import { logger } from './logger.js'
 
-/**
- * The ONLY custom bits needed for the router.
- *
- * Mount any custom endpoints, optionally reverse-proxying to the set of
- * underyling hosts with automatic failover, by using the injected revProxy handler
- */
+import { proxyWith } from './proxy.js'
+import { redirectWith } from './redirect.js'
 
-/**
- * The Reverse Proxy Configuration for an ao Compute Unit Router
- */
-function aoComputeUnitMount ({ app, revProxy }) {
-  app.get('/', revProxy({ processIdFromRequest: (req) => 'process' }))
-  app.get('/result/:messageTxId', revProxy({ processIdFromRequest: (req) => req.query['process-id'] }))
-  app.get('/results/:processId', revProxy({ processIdFromRequest: (req) => req.params.processId }))
-  app.get('/state/:processId', revProxy({ processIdFromRequest: (req) => req.params.processId }))
-  app.get('/cron/:processId', revProxy({ processIdFromRequest: (req) => req.params.processId }))
-  app.post('/dry-run', revProxy({ processIdFromRequest: (req) => req.query['process-id'] }))
+const middlewareWithByStrategy = {
+  proxy: proxyWith,
+  redirect: redirectWith,
```

Comment:
Dangling commas should be getting caught by the linter `standard` which runs on pre-commit. How's this getting through?

Reply:
I guess I never ran `npm install` at the monorepo root. Will fix.